templates/netlify/README.md

```diff
@@ -30,14 +30,24 @@ netlify init
 
 ## Development
 
-The Netlify CLI starts your app in development mode, rebuilding assets on file changes.
+The Remix dev server starts your app in development mode, rebuilding assets on file changes.
```

Comment:
[pebble] It may make sense to sequence the content in this section. Do users need to have the Remix App Server running before starting `netlify dev`?

Even if they don't, as currently written, we say the Remix server starts the app in dev mode. Then we way the Netlify CLI does the same thing. And we don't really define the relationship, so it could be interpreted as either or both running the app in a non-specific way.

```suggestion
To run your Remix app locally and properly test Netlify features, you need to:
1. Start the Remix App Server in development mode
2. Start Netlify Dev

### Start the Remix App Server in development mode

The Remix dev server starts your app in development mode, rebuilding assets on file changes. To start the Remix dev server:
```

Reply:
No in this case, when you run `netlify dev`, it will build the code to run the Remix app server in Netlify functions.